packages/admin/src/pages/build-application/[applicationId]/[sectionId]/question-type/add-word-count.page.tsx

```diff
@@ -0,0 +1,125 @@
+import { getApplicationFormSummary } from '../../../../../services/ApplicationService';
+import QuestionPageGetServerSideProps from '../../../../../utils/QuestionPageGetServerSideProps';
+import Meta from '../../../../../components/layout/Meta';
+import CustomLink from '../../../../../components/custom-link/CustomLink';
+import { Button, FlexibleQuestionPageLayout, TextInput } from 'gap-web-ui';
+import { ApplicationFormSection } from '../../../../../types/ApplicationForm';
+import { postQuestion } from '../../../../../services/QuestionService';
+import { getSummaryFromSession } from '../../../../../services/SessionService';
+import { QuestionWithOptionsSummary } from '../../../../../types/QuestionSummary';
+import ResponseTypeEnum from '../../../../../enums/ResponseType';
+import InferProps from '../../../../../types/InferProps';
+
+type RequestBody = {
+  maxWords: string;
+};
+
+export const getServerSideProps = async (
+  context: GetServerSidePropsContext
+) => {
+  const { applicationId, sectionId } = context.params as Record<string, string>;
+  const sessionCookie = getSessionIdFromCookies(context.req);
+
+  const applicationFormSummary = await getApplicationFormSummary(
+    applicationId,
+    sessionCookie
+  );
+
+  const { sectionTitle } = applicationFormSummary.sections.find(
+    (section) => section.sectionId === sectionId
+  ) as ApplicationFormSection;
+
+  async function fetchPageData() {
+    return {
+      backButtonHref: `/build-application/${applicationId}/${sectionId}/question-type`,
+      sectionTitle,
+    };
+  }
+
+  async function handleRequest(body: RequestBody, jwt: string) {
+    const questionSummary = (await getSummaryFromSession(
+      'newQuestion',
+      sessionCookie
+    )) as QuestionWithOptionsSummary;
+
+    const { optional, ...restOfQuestionSummary } = questionSummary;
+    const { maxWords: _, ...restOfBody } = body;
+
+    await postQuestion(jwt, applicationId, sectionId, {
+      ...restOfQuestionSummary,
+      ...restOfBody,
+      responseType: ResponseTypeEnum.LongAnswer,
+      validation: {
+        maxWords: body.maxWords,
+        mandatory: optional !== 'true',
+      },
+    });
+  }
+
+  return QuestionPageGetServerSideProps({
+    context,
+    fetchPageData,
+    onSuccessRedirectHref: `/build-application/${applicationId}/${sectionId}`,
+    jwt: sessionCookie,
+    onErrorMessage: 'Something went wrong while trying to load this page.',
+    handleRequest,
+  });
+};
+
+export default function AddWordCount({
+  fieldErrors,
+  formAction,
+  previousValues,
+  pageData: { sectionTitle, backButtonHref },
+  csrfToken,
+}: InferProps<typeof getServerSideProps>) {
+  return (
+    <>
+      <Meta
+        title={`${
+          fieldErrors.length > 0 ? 'Error: ' : ''
+        }Add a question - Manage a grant`}
+      />
+
+      <CustomLink
+        href={backButtonHref}
+        dataCy="cy_questionOptionsPage-backButton"
```

Comment:
can we not leave the page name out of the tag here? I don't think we have a use case for differentiating between back buttons on different pages

(also you should be able to get this in Cypress just by looking for an element with the text 'Back')

Reply:
Yeah fair - have removed